src/app/onion/dashboard/dashboard.component.ts

```diff
@@ -791,6 +792,16 @@ export class DashboardComponent implements OnInit, OnDestroy {
     this.changelogs = undefined;
   }
 
+   /**
+   * Opens the Change Log modal for a specified Learning Object and fetches the appropriate change logs
```

Comment:
I'm a little confused by this `sidePanelWatcher`. Is this JSDoc perhaps a copy and paste error - or am I missing something?

Is the watcher observing the side panel, or actively controlling it?

Reply:
That's certainly a copy and paste error.

The watcher both watches and controls. It does allow for direct manipulation interaction of the side panel (opening and closing from the host component) but it also allows the host component to monitor close events originating from within the side panel. I'm not attached to the name so if it's confusing we should change it.